src/app/containers/Header/__snapshots__/index.test.jsx.snap

```diff
@@ -475,19 +475,13 @@ exports[`Header should render correctly for WS frontPage 1`] = `
         class="c2"
         dir="ltr"
       >
-        Let us know say you agree to cookies
+        We don update our Privacy and Cookies Policy
```

Comment:
Why have these snapshots changed?

Reply:
🤔 not too sure, I'll look into it.

Reply:
![image](https://user-images.githubusercontent.com/30599794/67386265-865d5180-f58c-11e9-95da-ec43191b7fe8.png)
this is the consent banner behaviour without my changes ^


![image](https://user-images.githubusercontent.com/30599794/67386326-a7be3d80-f58c-11e9-8cdf-a9a0528949e9.png)
consent banner behaviour with my changes ^

Reply:
I believe it's something to do with me setting the attribute `secure:true`

Reply:
@thekp discovered that adding the `secure=true` attribute means that the cookie doesn’t get set on http and our localhost is http.

One way around this is to monkey patch the `js-cookie` set method so that we set it to false in our snapshot tests.

**in `src/app/containers/Header/index.test.jsx`**
```js
import Cookie from 'js-cookie';
const { set } = Cookie;

Cookie.set = (name, value, options) => {
  set(name, value, { ...options, secure: false });
};
```